db/models/donation.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Donation = db.define('donations', {
+  date: {
+    type: Sequelize.DATE,
+    defaultValue: Sequelize.NOW
+  },
+  amount: {
+    type: Sequelize.INTEGER, // in U.S. Cents
+    allowNull: false
+  },
+  method: {
+    type: Sequelize.ENUM('stripe', 'check', 'other')
+  },
+  thanked: {
+    type: Sequelize.BOOLEAN,
+    defaultValue: false
+  },
+  dedication: {
+    type: Sequelize.STRING
```

Comment:
I was thinking the dedication would be a pretty open-ended message string.  I was thinking we could have a list of donors with their custom messages. Although I'm realizing now that I don't have a field for the actual donor name lol